pkg/products/rhssocommon/reconciler.go

```diff
@@ -109,7 +109,7 @@ func (r *Reconciler) CleanupKeycloakResources(ctx context.Context, inst *integre
 		}
 	}
 
-	// Delete all clients
+	// Delete all clientsproducts/rhsso/reconciler.go:320
```

Comment:
Are these comments - taken as self notes? Should it be removed?

Reply:
Remove